label_studio/tasks/migrations/0048_failedprediction.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2.25 on 2024-08-12 21:22
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('projects', '0026_auto_20231103_0020'),
+        ('ml_models', '0009_alter_thirdpartymodelversion_provider'),
+        ('tasks', '0047_merge_20240318_2210'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='FailedPrediction',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('message', models.TextField(blank=True, default='', help_text='The message explaining why generating this prediction failed', null=True, verbose_name='message')),
+                ('error_type', models.CharField(default=None, help_text='The type of error that caused prediction to fail', max_length=512, null=True, verbose_name='error_type')),
+                ('model_version', models.TextField(blank=True, default='', help_text='A string value that for model version that produced the failed prediction. Used in both live models and when uploading offline predictions.', null=True, verbose_name='model version')),
+                ('created_at', models.DateTimeField(auto_now_add=True, verbose_name='created at')),
+                ('updated_at', models.DateTimeField(auto_now=True, verbose_name='updated at')),
```

Comment:
do we need updated_at if they are immutable?

Reply:
Yeah I was thinking about this as well, I think for now you are correct. I will remove